### DISCUSSION TITLE
v2.0 - PostgresML

### DISCUSSION THREAD

**Discussion by giscus** [postgresml/postgresml#529] [Announcements]
# v2.0 - PostgresML

Train and deploy models to make online predictions using only SQL, with an open source Postgres extension.

https://postgresml.org/user_guides/setup/v2/installation/

**Comment by dusanmarjanovic** [comment]
the command for installing postgres server dev files should be changed to this (postgres version in the end, otherwise it cant find a package):

sudo apt-get install -y \
    postgresql-server-dev-${POSTGRES_VERSION} \
    libpython3-dev \
    libclang-dev \
    cmake \
    pkg-config \
    libssl-dev \
    clang \
    build-essential \
    libopenblas-dev \
    python3-dev

**Comment by dusanmarjanovic** [comment]
also here i get an error with --version "0.4.5" but with "0.5.6" it works
export POSTGRES_VERSION=14
cp docker/Cargo.toml.no-python Cargo.toml && \
cargo install cargo-pgx --version "0.4.5" && \
cargo pgx init --pg${POSTGRES_VERSION} /usr/bin/pg_config && \
cargo pgx package

**Comment by zampaleu** [comment]
Trying to run dashboard:
cargo run:
error: error returned from database: relation "pgml.notebooks" does not exist.
Any idea?

**Reply by dusanmarjanovic** [reply]
do you have pgml schema and notebooks table in it?
did you provide a proper database url?
